JS/apicalls.js

```diff
@@ -0,0 +1,8 @@
+export const API_URL_base = "https://api.noroff.dev/api/v1";
+
+export const editPostUrl = API_URL_base + "/social/posts/" + localStorage.getItem("editPostId");
```

Comment:
This might work for you, but It is not a good idea to retrieve from local storage here. You can retrieve at the point of use that is get the editPostId when you want to use it

Reply:
changed to instead use query URL